examples/cpp/pyperf/PyPerfCollapsedPrinter.cc

```diff
@@ -97,6 +99,15 @@ void PyPerfCollapsedPrinter::processSamples(
           lostSymbols++;
         }
       }
+      if (sample.kernelStackId > 0) {
+        auto stacks = kernelStacks.get_stack_symbol(sample.kernelStackId, -1);
+        for (auto it = stacks.crbegin(); it != stacks.crend(); ++it) {
+          auto sym = *it;
+          std::fprintf(output_file, ";%s_[k]", sym.c_str());
+        }
+      } else if (sample.kernelStackId != -EFAULT) {
```

Comment:
Why this check? (also comment please)

Reply:
Uhh explained in commit. Adding a comment